### PR TITLE
DOCS-12208-network connection entitlements

### PR DESCRIPTION
**Pull request by luanamulesoft**

### CODE REVIEW

File: modules/ROOT/pages/tgw-about.adoc [hunk 1]
```diff
@@ -27,8 +27,8 @@ The following values are governed by AWS limits:
 * Maximum number of transit gateways per VPC
 * Maximum number of VPCs that you can attach to a transit gateway
 
-For information about limitations when attaching a VPC to a transit gateway, 
-see 
+For information about limitations when attaching a VPC to a transit gateway,
```
Comment (evinaku) [inline modules/ROOT/pages/tgw-about.adoc]: Can we please say "AWS transit gateway" here. When we talk about attaching to a Transit gateway, we need to call it AWS transit gateway.

File: modules/ROOT/pages/tgw-attach-arm.adoc [hunk 2]
```diff
@@ -20,7 +20,7 @@ Before you can add a transit gateway to CloudHub:
 See https://docs.aws.amazon.com/vpc/latest/tgw/tgw-getting-started.html[Getting started with transit gateways^] in the AWS documentation for information about creating a transit gateway on AWS.
 +
 One transit gateway can support up to 10 VPC attachments.
-* Purchase an Anypoint VPC offering with AWS Transit Gateway entitlements.
+* The number of transit gateways you can add depends on the network connections available to your Business Group.
```
Comment (evinaku) [inline modules/ROOT/pages/tgw-attach-arm.adoc]: The number of attachments you can make to AWS Transit Gateways depends on the network connections available to your Business Group.

File: modules/ROOT/pages/tgw-attach-arm.adoc [hunk 3]
```diff
@@ -79,13 +79,13 @@ The *Add transit gateway* page lists the steps to create a resource share in AWS
 image::tgw-add-overview.png[Add transit gateway page: Steps to create a resource share]
 +
 [NOTE]
-The MuleSoft AWS account ID that appears depends on your platform (US or EU).
+The MuleSoft AWS Account ID relevant for your Anypoint VPC is displayed at this step and it varies depending on your platform (US or EU). If the AWS configuration is performed by other teams or resources within your organization, please share this MuleSoft AWS account ID with them.
```
Comment (evinaku) [inline modules/ROOT/pages/tgw-attach-arm.adoc]: could we please remove "depending on your platform (US or EU)"

Comment (evinaku) [reply modules/ROOT/pages/tgw-attach-arm.adoc]: Maybe we do: "The MuleSoft AWS Account ID relevant for your Anypoint VPC is displayed at this step and it varies, so please use the Account ID displayed in this step for your AWS configuration. If the AWS configuration is performed by other teams or resources within your organization, please share this MuleSoft AWS account ID with them."

**Review by evinaku — COMMENTED** [review]
just had one comment

**Review by hannanelson — APPROVED** [review]

**Pull request merged**